src/plugins/console/public/plugin.ts

```diff
@@ -68,6 +69,7 @@ export class ConsoleUIPlugin implements Plugin<void, void, AppSetupUIPluginDepen
             docLinks: { DOC_LINK_VERSION, links },
           } = core;
 
+          await loadActiveApi(core.http);
```

Comment:
@yuliacech I could move this to `renderApp` in the application folder. Which I think would resolve the metrics limit since it would then be lazy loaded. BUT i would need to update `renderApp` to be async so I could await this. I want to ensure it finishes before we load the autocomplete class to avoid race conditions. 🤔